src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -37,18 +37,12 @@
 
 
     public BattleshipModel() {
-        computer_clipper = new Civilian("Computer_Clipper", 3, new Coordinate(5, 1), new Coordinate(5, 3));
-        computer_dinghy = new Civilian("Computer_Dinghy", 1, new Coordinate(10, 10), new Coordinate(10, 10));
-        computer_aircraftCarrier = new Military(false,"Computer_AircraftCarrier",5, new Coordinate(2,2),new Coordinate(2,6));
-        computer_battleship = new Military(true,"Computer_Battleship",4, new Coordinate(2,8),new Coordinate(5,8));
-        computer_submarine = new Military(true,"Computer_Submarine",2, new Coordinate(9,6),new Coordinate(9,7));
-
-        playerHits = new ArrayList<>();
-        playerMisses= new ArrayList<>();
-        computerHits = new ArrayList<>();
-        computerMisses= new ArrayList<>();
-
-        shipSquares = new ArrayList<>();
+        System.out.println("This is the parent BattleshipModel default constructor.");
```

Comment:
Can you elaborate what the benefit of instantiating the objects above is over instantiating them in the default constructor? Is is just better practice, or was it causing things to break? My guess is how the code was previously structured might play a role in the Hard firing bug, but you said in your PR that the bug was still there.